core/src/test/resources/spark-events/application_1555004656427_0144

```diff
@@ -0,0 +1,9 @@
+{"Event":"SparkListenerLogStart","Spark Version":"3.0.0-SNAPSHOT"}
```

Comment:
Could you add this file to `dev/.rat-excludes` to pass the Jenkins, @tgravescs ?